rules/promotions/external_alerts.toml

```
@@ -19,7 +19,7 @@ risk_score = 47
 rule_id = "eb079c62-4481-4d6e-9643-3ca499df7aaa"
 rule_name_override = "message"
 severity = "medium"
-tags = ["Elastic", "Windows", "APM", "Network", "macOS", "Linux"]
+tags = ["Elastic"]
```

```suggestion
tags = ["Elastic", "Windows", "APM", "Network", "macOS", "Linux"]
```
Changed to pass the tests.